File: Apps/AcademyGestorMobile/app/src/main/java/com/example/academygestormobile/Models/Curso.kt
package com.example.academygestormobile.Models

data class Curso(
    val id: Int? = null,
    val cod_curso: String = "",
    val nombre: String = "",
    val descripcion: String = "",
    val horario: String = "",
    val tipo: Tipo ,
    val activo: Byte? = null,

)

File: Apps/AcademyGestorDesktop/AcademyGestor/Models/Curso.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AcademyGestor.Modelos
{
    public class Curso
    {
        public int? id { get; set; }
        public string cod_curso { get; set; }
        public string nombre { get; set; }
        public string descripcion { get; set; }
        public string horario { get; set; }
        public Tipo tipo { get; set; }
        public byte? activo { get; set; }
        



    }
}
